pyproject.toml

```diff
@@ -102,3 +102,9 @@ ignore = ["D200", "D212"]
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/flowcept"]
```

Comment:
Why are you telling the build tool (hatch) what the package directory is? The src layout is automatically recognized by hatch and shouldn't need to be explicitly defined. Did you try installing without these lines?